htdocs/compta/facture/class/facture.class.php

```diff
@@ -2690,7 +2690,7 @@ public function delete($user, $notrigger = 0, $idwarehouse = -1)
 				return -5;
 			}
 
-			$sql = 'UPDATE '.MAIN_DB_PREFIX.'projet_task_time';
+			$sql = 'UPDATE '.MAIN_DB_PREFIX.'element_time';
 			$sql .= ' SET invoice_id = NULL, invoice_line_id = NULL';
 			$sql .= ' WHERE invoice_id = '.((int) $rowid);
```

Comment:
without sql migration this request fails

Reply:
Hi @frederic34 thank you very much for your help, i didn't thought about that ;)
I'm going to merge these 2 PR for this case